src/components/AssetHeader/WatchAssetButton.tsx

```diff
@@ -31,9 +31,9 @@ export const WatchAssetButton: React.FC<WatchAssetButtonProps> = ({ assetId, ...
       alignItems='center'
       minWidth='auto'
       borderRadius='full'
-      bg='var(--chakra-colors-background-button-secondary-base)'
+      bg='background.button.secondary.base'
       // eslint-disable-next-line react-memo/require-usememo
-      _hover={{ bg: 'var(--chakra-colors-background-button-secondary-hover)' }}
+      _hover={{ bg: 'background.button.secondary.hover' }}
```

Comment:
non-blocking: when there's an object prop, to prevent rerendering we normally either memoize it if its non-static, or break it out into a const at the top of the file if it is static (like in this case)

Reply:
gm https://github.com/shapeshift/web/pull/8020/commits/f6275d7d001e69b7a1c33ba9c080b7f0dc1ecfed